samples/cellular/nrf_cloud_coap_device_message/README.rst

```diff
@@ -0,0 +1,193 @@
+.. _nrf_cloud_coap_device_message:
+
+Cellular: nRF Cloud Device Message
+##################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF Cloud CoAP Device Message sample demonstrates how to use the `nRF Cloud CoAP API`_ to send `Device Messages <nRF Cloud Device Messages_>`_.
+Every button press sends a message to nRF Cloud.
+
+It also demonstrates use of the :ref:`lib_nrf_cloud_alert` and the :ref:`lib_nrf_cloud_log` libraries.
+The sample sends an alert when the device first comes online.
+It also sends a log message indicating the sample version, as well as when the button is pressed.
```

Comment:
```suggestion
It also sends a log message indicating the sample version when the button is pressed.
```
Did I get this right? Or does the log  message indicate also a button press?

Reply:
there is a counter for button presses that is printed